lib/domain/app_state/contact/get_contacts_state.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/app_state/initial.dart';
+import 'package:fluffychat/app_state/lazy_load_success.dart';
+import 'package:fluffychat/app_state/success.dart';
+import 'package:fluffychat/domain/model/contact/contact.dart';
+
+class GetContactsInitial extends Initial {
+  const GetContactsInitial() : super();
+
+  @override
+  List<Object?> get props => [];
+}
+
+class GetContactsLoading extends Success {
+  const GetContactsLoading() : super();
+
+  @override
+  List<Object?> get props => [];
+}
+
+class GetContactsSuccess extends LazyLoadSuccess<Contact> {
+  final String keyword;
+
+  const GetContactsSuccess({
+    required super.data,
+    required super.offset,
+    required super.isEnd,
+    required this.keyword,
+  });
+
+  @override
+  List<Object?> get props => [data, offset, isEnd, keyword];
+
+  GetContactsSuccess addMore(GetContactsSuccess other) {
+    return GetContactsSuccess(
+      data: data + other.data,
```

Comment:
`+` in list maybe don't give a good performance, can you change it to addAll

Reply:
can't use addAdd here, addAdd is a void func, it will change the data

Reply:
> + in list maybe don't give a good performance, can you change it to addAll

Can you show the evidence?